vpnmanager/vpnmanagerapi/src/main/java/com/kape/vpnmanager/api/data/model/JobError.kt

```diff
@@ -0,0 +1,20 @@
+package com.kape.vpnmanager.api.data.model
+
+/**
+ * Object containing the details of an API failure.
+ *
+ * @param code `JobErrorCode`.
+ * @param error `Error`.
+ */
+data class JobError(
+    val code: JobErrorCode,
+    val error: Error? = null,
```

Comment:
The Error is just for extra details in case the need arises. If you see the others `*Error` objects, they all have the same format, on some we use them, on some not. `JobError` derives from Throwable, and it's the object we use for failures on `Result<>`.